pypuf/experiments/experiment/logistic_regression.py

```diff
@@ -103,7 +103,7 @@ def analyze(self):
             self.combiner.__name__,
             self.learner.iteration_count,
             self.measured_time,
-            1.0 - tools.approx_dist(self.instance, self.model, min(10000, 2 ** self.n)),
+            0.0,  # accuracy kept for compatibility to old log files
```

Comment:
this is the approximate distance on all inputs, not just on the training set. We **must** keep this